Watchman.Discord/Areas/Protection/Services/AntiSpamService.cs

```diff
@@ -45,8 +45,8 @@ public async Task SetPunishment(Contexts contexts, Punishment punishment)
 
         private async Task MuteUserForSpam(Contexts contexts, TimeSpan length)
         {
-            var timeRange = new TimeRange(DateTime.Now, DateTime.Now.Add(length));
-            var muteEvent = new MuteEvent(contexts.User.Id, timeRange, "Spam detected (by bot)", contexts.Server.Id);
+            var timeRange = new TimeRange(DateTime.UtcNow, DateTime.UtcNow.Add(length));
```

Comment:
to utc nie ma znaczenia bo to i tak jest zapisywane w informacjach datetime
ale dobrze że poprawiłeś